docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,184 @@
+
+To enable CSI-assisted cloning into larger PVCs than the source PVC, the Storage Class used must have `allowVolumeExpansion` set to `true`.
+:::
+
+<br />
+
+### Setting up RBAC permissions for cloning across namespaces
+
+When the CDI performs a cloning operation, it runs under the `default` serviceaccount in the namespace of the target PVC. When the source PVC is in a different namespace, required permissions must be given to the serviceaccount. Version 4.2.0 of the VMO pack does this automatically, you can specify the namespaces for which the permissions are configured through the `vmEnabledNamespaces` option in the pack:
+
+```
+charts:
+  virtual-machine-orchestrator:
+
+    # This namespace will be used to store golden images
+    goldenImagesNamespace: "vmo-golden-images"
+
+    # These namespaces will be created and set up to deploy VMs into
+    vmEnabledNamespaces:
+      - "default"
+      - "virtual-machines"
+```
+
+<br />
+
+### Defining StorageProfiles to ensure CSI-assisted cloning
+
+While the CDI contains an internal database of CSIs and their supported access modes, volume modes and CSI-assisted cloning support, it is still recommended to create a `StorageProfile` to force these parameters to what we want them to be. For example:
+
+```
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: ceph-block
+spec:
+  claimPropertySets: 
+  - accessModes:
+    - ReadWriteMany
+    volumeMode: 
+      Block
+  cloneStrategy: csi-clone
+```
+
+The example above informs CDI that a Storage Class with the name `ceph-block` supports the `Block` volumeMode with the ReadWriteMany accessMode, and it should use `csi-clone` for cloning operations.
+
+More information about Storage Profiles is available in the [CDI documentation](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md).
+
+:::info
+If Storage Profiles are not manually defined, CDI will automatically generate one for every Storage Class, based on the information in its internal database. It is recommend to verify these parameters are accurate for use case and if not, overwrite them by defining the Storage Profile manually.
+:::
+
+When Storage Profiles are properly defined, you can optionally replace the target `pvc` sections in your `vmTemplate` resources with a `storage` section as noted in the [CDI documentation](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/datavolumes.md#storage):
+
+```
+apiVersion: spectrocloud.com/v1
+kind: VmTemplate
+metadata:
+  name: ubuntu-2204
+spec:
+  dataVolumeTemplates:
+    - metadata:
+        name: ubuntu-2204
+      spec:
+        source:
+          pvc: 
+            name: template-ubuntu-2204
+            namespace: vmo-golden-images
+        storage:
+          resources:
+            requests:
+              storage: 50Gi
+  template:
+    < rest of template >
```

Comment:
Please provide the rest of the template so that learners can copy this. The code preview box will limit how many lines will be displayed and create a scrollbar.